pkg/secretless/plugin/v1/doc.go

```diff
@@ -28,6 +28,13 @@ at startup by using the plugin directory flag:
 
   ./secretless-broker -p /etc/lib/secretless
 
+Optionally (and highly recommended) is also providing a checksum of plugins to the broker
```

Comment:
I think it would helpful here to note what kind of attack vectors we're protecting against.  That is, explain how an attacker without control of the secretless machine (ie, and hence unable to modify the checksum file itself) might be able to alter the plugin files.